config/bias.py

```diff
@@ -0,0 +1,4 @@
+config.dateObs="date"
+config.dateCalib="calibDate"
```

Comment:
Isn't `calibDate` the default? Why does it need overriding? (Same comment for `dark.py`, `flat.py`, `fringe.py`, below)